Penumbra/UI/ModsTab/ModPanelTagsFeatureTab.cs

```diff
@@ -0,0 +1,211 @@
+          + "If the mod already contains a local tag in its own tags, the local tag will be ignored.", _selector.Selected!.LocalTags,
+            out var editedTag, false);
+        _tutorial.OpenTutorial(BasicTutorialSteps.Tags);
+        if (tagIdx >= 0)
+            _modManager.DataEditor.ChangeLocalTag(_selector.Selected!, tagIdx, editedTag);
+
+
+        //Adds the buttons, right now the tags are just in a List that contains List<string>, I'd properly move the tags to a json file so its easier to add and remove tags, can also add a function so the users can add tags themselves
+        DrawButtons(_tagsList[0], "Type");
+        DrawButtons(_tagsList[1], "Genre");
+        DrawButtons(_tagsList[2], "Gear Types");
+        DrawButtons(_tagsList[3], "Gear Category");
+        DrawButtons(_tagsList[4], "Gear Styles");
+        DrawButtons(_tagsList[5], "Gear Material");
+
+    }
+
+
+    private void DrawButtons(List<string> tags, string title)
+    {
+        ImGui.Text(title);
+        //byte buttonsOnSameLine = 0;
+        foreach (var tag in tags)
+        {
+            //Checks if the mod does not have the tag
+            if (!_selector.Selected!.LocalTags.Contains(tag))
+            {
+                //Clicking the button adds the tag to the localTags list
+                if (ImGui.Button(tag))
+                {
+                    AddTag(tag, _selector.Selected!.LocalTags.Count);
+                }
+            }
+            //if it does we want to color the button so you can see its added, im pretty sure im using this wrong...?
+            else
+            {
+                var color = ImRaii.PushColor(ImGuiCol.Button, ColorId.SelectedCollection.Value());
+                //Clicking the button removes the tag from the localTags list
+                if (ImGui.Button(tag))
+                {
+                    RemoveTag(_selector.Selected!.LocalTags.IndexOf(tag));
+                }
+                color.Pop();
+            }
+            ImGui.SameLine();
+            //buttonsOnSameLine++;
+            //if (buttonsOnSameLine >= 8)
+            //{
+            //    ImGui.NewLine();
+            //    buttonsOnSameLine = 0;
+            //}
+            //else
+            //    ImGui.SameLine();
+        }
+        ImGui.Separator();
+    }
+
+    //Adds tag to the mods json
+    private void AddTag(string tag, int tagIdx)
+    {
+        _modManager.DataEditor.ChangeLocalTag(_selector.Selected!, tagIdx, tag);
+    }
+
+    //Removes the tag by passing an empty string
+    private void RemoveTag(int tagIdx)
+    {
+        _modManager.DataEditor.ChangeLocalTag(_selector.Selected!, tagIdx, "");
+    }
+
+    private static bool DebugTestButton(string name)
+    {
+        ImGui.Button("TEST: " + name);
+        return ImGui.IsItemClicked();
+    }
+
+    //This is just a hacky test
+    private void AddTagsToList()
```

Comment:
Yea I've been thinking about that as well and I guess you're right (would also be benefitial for those that use another language than english) but it was mostly just, as you say so yourself, to keep some kind of consistency across all users but I guess non-programmers just don't get that 😁 

So somekind of "edit mode" where you have the option to add categories and in those categories you can add/delete/move tags by using drag and drop? Also do you think it would be better to use a list element instead of all the buttons? I guess I'll have to test some options to see what works best

Do you have an example of a "manager"?